src/components/StartingPage.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta
+			http-equiv="X-UA-Compatible"
+			content="IE=edge"
+		/>
+		<meta
+			name="viewport"
+			content="width=device-width, initial-scale=1.0"
+		/>
+		<link
+			rel="stylesheet"
+			href="../styles/root.css"
+		/>
+		<title>Quiz App</title>
+
+		<style
+			src="./styles/root.css"
+			rel="stylesheet"
+		></style>
+	</head>
+	<body>
+		<div
+			class="padding-small-card shade classA translateOne"
+		>
+			<img
+				class="quiz-index-img"
+				src="./img/1_jRuU1_OQ1P98SRjniHx1Jg.jpeg"
+				alt="quiz-img"
+			/>
+			<div
+				class="padding-small-card shade classA translateOne container-absolule quiz-container-bg col-6"
+			>
+				<div
+					class="quiz-container padding-small-card shade imgTransition col-12"
+				>
+					<img
+						class="quiz-img"
+						src="./img/movies collage.jpeg"
+						alt="quiz1-Img"
```

Comment:
suggstion: iin the alt you can avoid writing img, as it is already given that its an image and in the case it fails to load, the alt tag will tell us what it was supposed to be. So, please use a descriptive alt tag as well.

Reply:
I'll do that